src/Controller/HistoriesController.php

```diff
@@ -79,7 +79,10 @@ public function add()
             'valueField' => 'username',
             'limit' => 200
         ]);
+        $time = Time::now('Europe/Paris');
```

Comment:
Ez egy hardkódolt érték ami nem egészséges. Ilyesmiket jobb lenne egy config fileba tenni (akár az app.php-ba), vagy az AppControllerben beírni a config értékek közé és onnna kiolvasni